server/src/__tests__/__utils.js

```diff
@@ -0,0 +1,29 @@
+const {HttpLink} = require('apollo-link-http');
```

Comment:
Is there a reason why we're using __tests__ as a folder name? Why not just tests?

Reply:
This is apollo.server convention.